packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -12,58 +12,58 @@
 
 .root {
   margin-block-start: 0;
-  margin-block-end: $spacing-xs-em;
+  margin-block-end: var(--SpaceMarginBlockEnd);
   margin-inline: 0;
-  color: $text-heading;
-  font-family: $body-font-family;
-  font-size: $base-font-size;
+  color: var(--ColorTextHeading);
+  font-family: var(--FontFamilyTitle);
+  font-size: var(--FontSizeBase);
   font-weight: 600;
 }
 
 .level1 {
-  font-size: $size-title-1;
+  font-size: var(--SizeTitle1);
 }
 
 .level2 {
-  font-size: $size-title-2;
+  font-size: var(--SizeTitle2);
 }
 
 .level3 {
-  font-size: $size-title-3;
+  font-size: var(--SizeTitle3);
 }
 
 .level4 {
-  font-size: $size-title-4;
+  font-size: var(--SizeTitle4);
 }
 
 .level5 {
-  font-size: $size-title-5;
+  font-size: var(--SizeTitle5);
 }
 
 .level6 {
-  font-size: $size-title-6;
+  font-size: var(--SizeTitle6);
 }
 
 .level1,
 .level2,
 .level3 {
-  line-height: $title-line-height;
+  line-height: var(--FontLineHeightTitlegit);
```

Comment:
ugh this is a tough one to name, lets avoid the use of `Title` here as we're refactoring away from it.

Reply:
cc @arnoldsandoval-okta still seeing the word `Title` throughout this PR. can you adjust prior to merge here?

Reply:
bump ☝️

Reply:
@arnoldsandoval-okta any movement on this PR?

Reply:
Has been updated.